### PR TITLE
Add site setting to ask new user to add family member

### PR DESCRIPTION
**Pull request by deltaisfordeath**

### CODE REVIEW

File: src/GRA.Controllers/ViewComponents/DisplayNotificationsViewComponent.cs [hunk 1]
```diff
@@ -77,6 +77,10 @@ public IViewComponentResult Invoke()
                         notification.AttachmentFilename
                             = _pathResolver.ResolveContentPath(notification.AttachmentFilename);
                     }
+                    if (HttpContext.Items[ItemKey.AskToAddFamilyMember].ToString() == "True")
+                    {
+                        notification.SupplementalText = $"Would you like to <a href='{Url.Action(nameof(ProfileController.AddHouseholdMember), ProfileController.Name)}'>add a family member?</a>";
```
Comment (k7hpn) [inline src/GRA.Controllers/ViewComponents/DisplayNotificationsViewComponent.cs:82]: This needs to be localized properly.

**Review by k7hpn — CHANGES_REQUESTED** [review]
Just need to localize that text.

File: src/GRA.Controllers/ViewComponents/DisplayNotificationsViewComponent.cs (hunk 1 above)
Comment (deltaisfordeath) [reply src/GRA.Controllers/ViewComponents/DisplayNotificationsViewComponent.cs:82]: I thought as much. FYI there are other notifications that go through that component that aren't getting localized currently:
Screenshot 2024-01-09 164725
Screenshot 2024-01-09 164850

Comment (k7hpn) [reply src/GRA.Controllers/ViewComponents/DisplayNotificationsViewComponent.cs:82]: Opened an issue for these two:  #1082

**Pull request merged**